README.md

```diff
@@ -90,24 +91,22 @@ mvn spring-boot:stop
 #
 # make sure you are in the root of the repo
 
-# Java Team - is the below line correct or should it be KEYVAULT_NAME?
-# $He_Name is set to the name of your key vault
 # this will use Azure CLI cached credentials
 
 # build the image
 
-docker build . -t helium-dev .
+docker build -t helium-dev .
+
+# Then run the container.  
 
-# run the container
+# First obtain APPLICATIONINSIGHTS_CONNECTION_STRING using the following link:
+#TEST THIS
+https://docs.microsoft.com/en-us/azure/azure-monitor/app/java-get-started?tabs=maven
 
-# mount your ~/.azure directory to container root/.azure directory
-# you can also run the container and run az login from a bash shell
+export APP_INSIGHTS_CONNECTION_STRING=<KEY_FROM_ABOVE_LINK>
```

Comment:
Should we leave this out for simplification?

Reply:
Yes thank you we took it out and modified the docker run command to do the mount:
docker run -p4120:4120 --env AUTH_TYPE=CLI --env KEYVAULT_NAME=$KEYVAULT_NAME --env APPLICATIONINSIGHTS_CONNECTION_STRING=$APP_INSIGHTS_CONNECTION_STRING -v ~/.azure:/home/helium/.azure helium-dev:latest